client/signup/jetpack-connect/index.jsx

```diff
@@ -180,7 +184,7 @@ const JetpackConnectMain = React.createClass( {
 					<p>{ this.translate( 'We need to send you to your WordPress install so you can approve the Jetpack installation. Click the button in the bottom-right corner on the next screen to continue.' ) }</p>
```

Comment:
Not related to this particular PR, but... Is "your WordPress install" correct? I'm thinking "WordPress installation" or "WordPress Plugins admin page". Just wondering, don't really know  if "WordPress install" is an usual expression or not